src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerData.cs

```diff
@@ -1212,19 +1211,25 @@ private void LoadGeneralProperties()
                 this.reservedStorageSizeMB = server.ReservedStorageSizeMB;
                 this.storageSpaceUsageInMB = server.UsedStorageSizeMB;
             }
-            else
+            if (server.VersionMajor < 14 || server.HostPlatform == null)
             {
-                this.isXTPSupported = server.IsXTPSupported;
+                this.platform = server.Platform;
+                this.operatingSystem = "Microsoft Windows NT " +
+                        Convert.ToString(server.OSVersion, System.Globalization.CultureInfo.InvariantCulture);
             }
-            if (server.VersionMajor >= 14)
+            else
             {
-                this.operatingSystem = server.HostDistribution;
                 this.platform = server.HostPlatform;
+                this.operatingSystem = string.Format("{0} ({1})", server.HostDistribution, server.HostRelease);
```

Comment:
Interpolated strings are great 😄 

```suggestion
                this.operatingSystem =$"{server.HostDistribution} ({server.HostRelease})";
```